RELEASE_NOTES.md

```diff
@@ -1,4 +1,8 @@
-# Next Version 1.0.19 (2018-01-22)
+# Development 1.0.20 (2018-01-22)
+
+- Issue #11419 - Added table ctdav_auv_data for the new Neil Brown CTD.
+
+# Version 1.0.19 (2018-01-22)
```

Comment:
has version 1.0.19 been released?  latest tag is listed in github as 1.0.17.  the dates seem out of wack.

Reply:
Tim had expressed in the last Sprint that he would like one version per ticket.  (However, I see I have to update the date on the version I added at the top ... oops)